src/panels/config/core/ha-config-section-general.ts

```diff
@@ -26,6 +26,8 @@ import "../../../layouts/hass-subpage";
 import { haStyle } from "../../../resources/styles";
 import type { HomeAssistant } from "../../../types";
 import "../../../components/ha-alert";
+import { showConfirmationDialog } from "../../../dialogs/generic/show-dialog-box";
+import type { HaCheckbox } from "../../../components/ha-checkbox";
```

Comment:
You only import the type, not the element itself
```suggestion
import type { HaCheckbox } from "../../../components/ha-checkbox";
import "../../../components/ha-checkbox";
```